Gruntfile.coffee

```diff
@@ -1227,11 +1208,15 @@ module.exports = (grunt) ->
     'jasmine_chromeapp:socksEcho'
   ]
 
+  grunt.registerTask 'unit_test_nobuild', _.flatten(
```

Comment:
FYI, this line is missing [] around the list to be flattened. As a result, flatten is running only on the first list.

The correct code would be:
````
  grunt.registerTask 'unit_test_nobuild', _.flatten([
    Rule.buildAndRunTest(spec, grunt) for spec in Rule.getTests('src', 'lib', ['build-tools', 'integration-tests']),
    Rule.buildAndRunTest(spec, grunt) for spec in Rule.getTests('src', 'generic_core'),
    Rule.buildAndRunTest(spec, grunt) for spec in Rule.getTests('src', 'generic_ui/scripts')
  ])
````

I'll send a PR that will fix that.